pkg/projects/netcoreappRIDs.props

```diff
@@ -34,7 +31,16 @@
     <OfficialBuildRID Include="linux-arm">
       <Platform>arm</Platform>
     </OfficialBuildRID>
+
+    <!-- The following RIDs are not officically supported and are not 
+         built during official builds, however we wish to include them 
+         in our runtime.json to enable others to provide them.  -->
+    <OfficialBuildRID Include="debian.8-armel">
```

Comment:
Do you think it would be clearer if I renamed these items to something like "UnofficialBuildRID" or "UnsupportedBuildRID"?